src/Microsoft.Extensions.Logging.Console/ConsoleLogger.cs

```diff
@@ -107,11 +107,10 @@ public virtual void WriteMessage(LogLevel logLevel, string logName, int eventId,
         {
             var logLevelColors = default(ConsoleColors);
             var logLevelString = string.Empty;
-            var logIdentifier = string.Empty;
-            var scopeInformation = string.Empty;
-            var exceptionText = string.Empty;
             var printLog = false;
 
+            var sb = new StringBuilder();
```

Comment:
We could consider using a threadlocal to cache these if we need to. Logging is 99.99% likely to be non-reentrant.